src/components/shared/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FunctionComponent } from 'react';
+import styled from 'styled-components';
+
+interface HeaderProps {
+    children: string;
+}
+
+const Header: FunctionComponent<HeaderProps> = ({ children }) => (
+    <div className="header">
```

Comment:
Whether you use a styled component or not, you can probably get out of the habit of using `className`.
```suggestion
    <div>
```